src/core/utils/index.js

```diff
@@ -336,3 +348,39 @@ export const getAddonTypeFilter = (addonType, _config = config) => {
 
   return ADDON_TYPE_THEMES_FILTER;
 };
+
+export const getExtensionTypeAction = (type = ADDON_TYPE_EXTENSION) => {
+  let action = TRACKING_TYPE_EXTENSION;
+
+  if (type === ADDON_TYPE_STATIC_THEME) {
+    action = TRACKING_TYPE_STATIC_THEME;
+  }
+
+  return action;
+};
+
+export const getAddonCategory = ({
+  type = ADDON_TYPE_EXTENSION,
+  installCategory = INSTALL_CATEGORY,
+} = {}) => {
+  let category;
+  const isTheme = ADDON_TYPE_THEMES.includes(type);
+
+  switch (installCategory) {
+    case INSTALL_STARTED_CATEGORY:
```

Comment:
I find the naming and use of these constants confusing. This constant `INSTALL_STARTED_CATEGORY` is actually only used here as a switch, so it could contain a value like `install:started`, but it in fact contains the value `AMO Addon / Theme Installs Started` even though that's never used as an actual category (because the actual category will be either `INSTALL_THEME_STARTED_CATEGORY` or `INSTALL_EXTENSION_CATEGORY`).

I think it would make sense to create a new set of constants with names like `INSTALL_STARTED_ACTION`, which contain values like `install:started`, and use those. The constant names using `*_CATEGORY` are being overloaded and it's making it quite confusing to follow.

Reply:
thanks @bobsilverberg ...I'll take another look at this.